example/admin/config.yml

```diff
@@ -155,3 +155,14 @@ collections:
         hint: "An int64 value."
         widget: "number"
         value_type: "int"
+
+      - name: "custom_value"
+        label: "CUSTOM VALUE"
+        comment: "A value with a custom widget."
+        required: false
+        hint: "A value with a custom widget."
+        widget: "test"
+        bar: 'baz'
+        foo: 5
```

Comment:
Those might be needed to be removed later?

I am fine merging as it is and later I can find time to help clean it up as well.